config/default.yaml

```diff
@@ -270,6 +270,8 @@ ecosystem:
   ui: https://cd.screwdriver.cd
   # Externally routable URL for the Artifact Store
   store: https://store.screwdriver.cd
+  # Externally routable URL for the Queue Service
+  queue: ECOSYSTEM_QUEUE
```

Comment:
Would this be an actual address like  `https://queue.screwdriver.cd` ? Since `config/custom-environment-variables.yaml` already declared as environment variable.

Reply:
Yeah, we need a similar endpoint for the queue.